src/electionguard/guardian.py

```diff
@@ -41,16 +42,41 @@
 from .types import GUARDIAN_ID
 from .utils import get_optional
 
+
+@dataclass
+class GuardianRecord(Serializable):
+    """Published record of Guardian"""
```

Comment:
as i mentioned on the call, would be good to have more of a description here so that i can know the context of a guardian record without having to go look it up in the docs